contentcuration/contentcuration/frontend/channelEdit/views/ImportFromChannels/BrowsingCard.vue

```diff
@@ -192,30 +181,48 @@
         return get(this.node, ['description', 'length']) > 120;
       },
       numLocations() {
-        return 1;
+        return this.node.location_ids.length;
       },
       goToLocationLabel() {
         return this.numLocations > 1
           ? this.$tr('goToPluralLocationsAction', { count: this.numLocations })
           : this.$tr('goToSingleLocationAction');
       },
-      goToLocationRoute() {
-        return {
-          name: this.RouterNames.IMPORT_FROM_CHANNELS_BROWSE,
-          params: {
-            channelId: this.node.source_channel_id,
-            nodeId: this.node.parent,
-          },
-          query: {
-            nodeId: this.node.id,
-          },
-        };
+      isTopic() {
+        return this.node.kind === 'topic';
       },
     },
     methods: {
-      emitPreview() {
-        this.$emit('preview');
+      ...mapActions('clipboard', ['copy']),
```

Comment:
`BrowsingCard` is a small component that hadn't been connected to the store before these updates. Introducing such a tight dependency only because of clipboard seems unnecessary - instead, I would suggest moving that logic to its parent views that are more complex and already connected to the store (`ContentTreeList`, `SearchResultsList`). To avoid having `handleClickClipboard` two times in those views, maybe it can be moved to the store? It looks quite general. In this case, it doesn't look like a blocker to me. It can be done as a follow-up or skipped for now, just wanted to point it out because, in general, these dependencies are something to be careful about - we'll have issues testing and refactoring later if there's too much of them.